docs/modules/ROOT/pages/how-to-guides/testing_applications/proc_retriggering_integration_tests.adoc

```diff
@@ -0,0 +1,35 @@
+= Retriggering Integration Tests
+
+Integration test scenarios for a given snapshot can be re-triggered by adding a label to the snapshot.
+
+== Prerequisites
+- The snapshot for which tests will be retriggered has completed its initial tests
+- CLI access to the OpenShift cluster in question.  For information on getting access to the CLI, see xref:../../getting-started/getting_started_in_cli.adoc[Getting started in CLI]
+- An up-to-date kubectl binary.  The `oc` binary may be used as well.
+
+== Procedure
+. Determine which IntegrationTestScenario should be re-run.  For a given snapshot, only one scenario can
+be re-run at a time.
+. Add the label `test.appstudio.openshift.io/run` to the snapshot. Set the value to the name of the scenario to be re-run
+[source]
+$ kubectl label [snapshot name] test.appstudio.openshift.io/run=[scenario name]
+
+. The tests will be re-triggered automatically.  Once they are re-triggered the label will be removed.  At this point you
+may add the label again with a different scenario if you wish to re-test multiple scenarios.
+[source]
+$ tkn pipelinerun list                                                                                                                                                          main 
+NAME                            STARTED         DURATION   STATUS
+integration-pipelinerun-jfrdb   4 seconds ago   ---        Running
+
+. If you run `tkn pipelinerun describe [pipelinerun]` you can see that the `appstudio.openshift.io/snapshot` and `test.appstudio.openshift.io/scenario` labels reference the correct snapshot and scenario.
```

Comment:
```suggestion
. To verify the tests, run `tkn pipelinerun describe [pipelinerun]` to ensure that the labels `appstudio.openshift.io/snapshot` and `test.appstudio.openshift.io/scenario` correctly reference the correct snapshot and scenario.
```

Reply:
I tweaked this suggestion a bit, let me know if its fine this way.